reports/BUILD

```diff
@@ -0,0 +1,9 @@
+py_library(
+    name = "reports",
+    srcs = [
+        "__init__.py",
+        "overall.py",
+        "diversification.py"
+    ],
```

Comment:
dependencies are missing for pandas and marketplace

Reply:
I'm going to add